_includes/common/ui/transactions-page.md

```diff
@@ -21,18 +21,23 @@ To view this page, [access the DB Console](ui-overview.html#db-console-access) a
 To view this page, click **SQL Activity** in the left-hand navigation of the {{ site.data.products.db }} Console and then click the **Transactions** tab.
 {% endif %}
 
-## Search and filter by application
+## Search and filter
 
-By default, this page shows transactions from all applications running on the cluster, and hides internal CockroachDB transactions.
+By default, this page shows transactions from all applications and databases running on the cluster.
```

Comment:
we also hide internal transactions by default

Reply:
That's addressed in line 32: When no application is selected internal transactions **are not** displayed.

Reply:
sounds good